src/test/java/org/jitsi/meet/test/MalleusJitsificus.java

```diff
@@ -48,6 +49,12 @@
         = "org.jitsi.malleus.duration";
     public static final String ROOM_NAME_PREFIX_PNAME
         = "org.jitsi.malleus.room_name_prefix";
+    public static final String INTERVAL
```

Comment:
could you name this something a bit more descriptive?  it's not clear to me from reading them what they mean.